pyproject.toml

```diff
@@ -22,6 +22,9 @@ dependencies = [
     "scanspec",
     "PyYAML",
     "click",
+    "fastapi",
+    "uvicorn",
+    "httpx",
```

Comment:
What's this for?

Reply:
for the fastAPI application to run it needs these three dependencies.